client/src/components/EditDetails.jsx

```diff
@@ -64,75 +72,144 @@ class EditDetails extends React.Component {
             gender:"${gender}",
             dob:"${dob}",
             zip:${zip}
-          ) { id } 
-        }`;
-      axios
-        .post('/ct/graphql', {query: mutation})
-        .then(() => console.log('details editted'))
-        .catch(err => console.error(err));
-    }
+          ) { id }
+        }`
+
+    axios
+      .post('/ct/graphql', { query: mutation })
+      .then(() => {
+        console.log('details editted')
+        window.location.reload()
```

Comment:
I moved `reload()` here, because at the original place, it seems like reload will be called too early that the rest of the code could not have time to execute, which potentially introduces a buggy [race condition](https://searchstorage.techtarget.com/definition/race-condition)